### DISCUSSION TITLE
Error when executing a query with LIMIT through doris-spark-connector 1.3.1

### DISCUSSION THREAD

**Discussion by SongboWang1998** [apache/doris#39518] [A - General / Q&A] [answered]
Hello Doris Community,

I am encountering an issue when executing SQL queries that include a LIMIT clause through the doris-spark-connector. The same queries work fine when executed directly in the Doris client, but they fail when run through the connector.

**Error Message:**
ERROR DorisBatch: Doris FE's response cannot map to schema. res: {"exception":"only support single table filter-prune-scan, but found [ select user_id, city from xxx LIMIT 1]","status":400} org.apache.doris.shaded.com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "exception" (class org.apache.doris.spark.rest.models.QueryPlan), not marked as ignorable (3 known properties: "partitions", "status", "opaqued_query_plan"]) at [Source: (String)"{"exception":"only support single table filter-prune-scan, but found [ select user_id, city from xxx LIMIT 1]","status":400}"; line: 1, column: 15] (through reference chain: org.apache.doris.spark.rest.models.QueryPlan["exception"])

This error suggests that the query is not supported for filter-prune-scan operations on a single table when using the doris-spark-connector, even though it works directly in the Doris client. Additionally, there seems to be an issue with parsing the error response in the connector.

**Steps to Reproduce:**
1. Execute the query `SELECT user_id, city FROM xxx LIMIT 1;` directly in the Doris client (works fine).
2. Execute the same query through the doris-spark-connector (results in the above error).

**Environment:**
- Doris version: 2.0.13
- Spark Doris Connector version: 1.3.1
- Apache Spark version: 3.5

Could anyone please help me understand why this discrepancy occurs and how to resolve it? Additionally, any advice on handling the unrecognized field issue in the error response would be greatly appreciated.

Thank you for your assistance.

**Comment by ixzc** [comment, marked as answer]
this is not supported

**Reply by smallmonkey1019** [reply]
How to solve it
